pkg/metamorph/client_test.go

```diff
@@ -204,12 +203,12 @@ func TestClient_SubmitTransaction(t *testing.T) {
 
 			require.Equal(t, tc.expectedStatus, status)
 
-			if tc.expectedErrorStr == "" {
+			if tc.expectedErrorStr != "" {
 				require.NoError(t, err)
```

Comment:
Wondering why the test passes, but I think `require.NoError(t, err)` needs to be removed here

Reply:
Removed